tests/vmi_multus_test.go

```diff
@@ -1029,6 +1042,19 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 	return setUpInterface(vmi, interfaceName)
 }
 
+func configSRIOVInterface(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, interfaceMac, interfaceAddress string) error {
```

Comment:
Maybe changing this function name to`configureInterfaceStaticIPByMAC` will describe better what it does
 what you think?

Reply:
I like long informative names.
changed to: 
configureSRIOVInterfaceStaticIPByMAC